util/env/env_test.go

```diff
@@ -7,71 +7,133 @@ import (
 
 func TestSplitEnvToListWithColon(t *testing.T) {
 	os.Setenv("TEST_LIST", "foo,bar")
-	list := SplitEnvToList("TEST_LIST")
+	list := SplitEnvToDeps("TEST_LIST")
 	if list == nil {
 		t.Errorf("Expected: not nil")
 	}
-	if list[0] != "foo" {
+	if list[0].Name != "foo" {
```

Comment:
@dkrzyszczyk why isn't this test using ginkgo? You should clean up after tests and delete all the environmental variables you set (or just re-set them to original value).

Reply:
@dkrzyszczyk I was told that tests use two different approaches anyway. You can ignore part of my comment related to ginkgo.

Reply:
@dkrzyszczyk use `defer os.Unsetenv("whatever-was-the-variable-name")` at the beginning of each function that sets any evironmental variables. Each of them should be unset :)